examples/mnist.py

```diff
@@ -249,6 +249,7 @@ def main(model='mlp', num_epochs=500):
         network = build_cnn(input_var)
     else:
         print("Unrecognized model type %r." % model)
+        sys.exit(1)
```

Comment:
Oh yes, that was missing. I'd prefer to just have a `return` here. If someone imports the file and calls `main()`, they wouldn't want their program to exit if `model` was unrecognized. Do you agree? Could you amend your commit to do a `return` instead of `sys.exit(1)`? (Edit the file, do `git add mnist.py; git commit --amend; git push --force`.) Thanks!